deployments/container/reconfigure-mig.sh

```diff
@@ -525,27 +541,36 @@ if [ "${?}" != "0" ]; then
 fi
 
 if [ "${CDI_ENABLED}" = "true" ]; then
-	echo "Running nvidia-smi"
-	chroot ${DRIVER_ROOT_CTR_PATH} nvidia-smi >/dev/null
-	if [ "${?}" != "0" ]; then
-		exit_failed
+	if [ "${DRIVER_ROOT_CTR_PATH}" = "/host" ]; then
```

Comment:
Does the `/run/nvidia/driver` case still work as expected?

Reply:
Base on my initial testing, yes. But will keep investigating this.

Reply:
I was wrong -- the `/run/nvidia/driver` case was not working with this change. As suspected, without running `nvidia-smi` the `nvidia-cap*` device nodes were note created / updated correctly after a MIG configuration, and thus, the CDI specifications were not accurate. I have raised https://github.com/NVIDIA/mig-parted/pull/82 to address both the `/run/nvidia/driver` case as well as the GKE case, where driverRoot != devRoot.